Doc/howto/functional.rst

```diff
@@ -1208,8 +1208,8 @@ General
 -------
 
 **Structure and Interpretation of Computer Programs**, by Harold Abelson and
-Gerald Jay Sussman with Julie Sussman.  Full text at
-https://mitpress.mit.edu/sicp/.  In this classic textbook of computer science,
+Gerald Jay Sussman with Julie Sussman.  The book can be found at
+https://mitpress.mit.edu/sicp.  In this classic textbook of computer science,
```

Comment:
The MIT Press website just says that the book is out of print -- I found [another resource](https://mitp-content-server.mit.edu/books/content/sectbyfn/books_pres_0/6515/sicp.zip/index.html) on the MIT domain, but I don't know how long-term the link will be.

Reply:
There is also wayback machine (long term available) to get the point in history where the book was freely available. I didn't linked it before as I thought it wouldnt be appropriate, considering they are selling it now 🤷‍♂️